docs/install/1-install-server.rst

```diff
@@ -4,7 +4,7 @@ Install Inmanta
 ***************
 
 This page explains how to install the Inmanta orchestrator software and setup an orchestration server. Regardless what platform
-you installed it on, Inmanta requires at least the latest Python 3.9 and git to be installed.
+you installed it on, Inmanta requires git to be installed.
```

Comment:
I think this can go since inmanta service orchestrator server installs python 3.11